.travis.yml

```diff
@@ -37,7 +38,7 @@ jobs:
     - stage: build-package
       name: Building package & Deploying to Storage Account
       sudo: required
-      if: ((branch = master OR type = pull_request) AND fork = false) OR tag =~ /^release.*$/
+      if: branch = master OR tag =~ /^release.*$/
```

Comment:
it should run on all branches not only on master if you remove PR

Reply:
Done.